integration-tests/opentelemetry-vertx/src/main/resources/reflection-config.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Is this needed because of how we're testing?

Reply:
Yes. This is in the integration test modules, so not a big deal.